CRM/Price/Form/Preview.php

```diff
@@ -132,7 +132,9 @@ public function buildQuickForm() {
         foreach ($group['fields'] as $field) {
           $fieldId = $field['id'];
           $elementName = 'price_' . $fieldId;
-          CRM_Price_BAO_PriceField::addQuickFormElement($this, $elementName, $fieldId, FALSE, $field['is_required']);
+          if (!empty($field['options'])) {
+            CRM_Price_BAO_PriceField::addQuickFormElement($this, $elementName, $fieldId, FALSE, $field['is_required']);
+          }
```

Comment:
I would have thought so as we don't want fields with disabled FTs showing publicly but we want them able to be accessible on the backend so admins can alter the FT